pkg/controller/dataimportcron-controller.go

```diff
@@ -1036,6 +1024,21 @@ func addDataImportCronControllerWatches(mgr manager.Manager, c controller.Contro
 		return err
 	}
 
+	mapCronJobToCron := func(_ context.Context, obj client.Object) []reconcile.Request {
+		return []reconcile.Request{{NamespacedName: types.NamespacedName{Namespace: obj.GetLabels()[common.DataImportCronNsLabel], Name: getCronName(obj)}}}
```

Comment:
hmm may want to back out if `obj.GetLabels()[common.DataImportCronNsLabel]` is nil? not sure that's possible but could get messy in the reconcile loop